### DISCUSSION TITLE
What happens to builds in the obs devel project now?

### DISCUSSION THREAD

**Discussion by badshah400** [openSUSE/openSUSE-git#41] [Q&A] [answered]
I am trying to learn the ropes os using src.o.o and using the [obs://devel:libraries:c_c++/armadillo](https://build.opensuse.org/package/show/devel:libraries:c_c++/armadillo) package for my tests. I have managed to update the package version in Factory [see [sr#1183947](https://build.opensuse.org/request/show/1183947)] using the git workflow. However, this seems to have [broken builds](https://build.opensuse.org/package/live_build_log/devel:libraries:c_c++/armadillo/openSUSE_Factory/x86_64) in the obs devel project itself, with weird errors, that nonetheless did not hinder the package's acceptance into oS:Factory. Are builds in the devel project supposed to be disabled now or am I doing something wrong here?

Thanks in advance for any advice.

**Comment by dcermak** [comment, marked as answer]
I doubt that you actually broke the build, the whole devel project is red for Tumbleweed: https://build.opensuse.org/project/monitor/devel:libraries:c_c++?arch_x86_64=1&defaults=0&failed=1&repo_openSUSE_Factory=1

And no, you should not disable builds in the devel project.

**Reply by badshah400** [reply]
Ah, I see. Many thanks.

Also it would be helpful to know how the [open PR](https://src.opensuse.org/pool/armadillo/pulls/1) gets merged. That did not happen automatically when the package was accepted into oS:Factory. Shouldn't it be automated somehow? It seems I cannot do the merge either (though I am the maintainer for the package in the obs devel project).

**Reply by badshah400** [reply]
I see that this was merged by src-o-org-forwarder already.
